external/mcpclient/src/mcp-client-plugin.ts

```diff
@@ -3,16 +3,48 @@ import { Client, ClientOptions } from '@modelcontextprotocol/sdk/client/index.js
 import { SSEClientTransport } from '@modelcontextprotocol/sdk/client/sse.js';
 import { Transport } from '@modelcontextprotocol/sdk/shared/transport.js';
 
-export type McpClientPluginParams = {
+type McpClientToolDetails = {
```

Comment:
any types publicly visible via class methods/constructors/properties should be exported publicly as well.